WordPress/playground-tools
# WIP wp-now: update blueprints dependencies

This PR is a draft WIP.



- Related to https://github.com/WordPress/playground-tools/issues/113

## What?



Updates Playground dependencies

## Why?



 `@wp-playground/blueprints` v0.2.0 removes the dependency on `DOMParser` for the `activatePlugin` blueprint step.


## Testing Instructions
- Run `nvm use`
- `npm install`
- `npx nx build wp-now`
- Create the blue print file `b.json`
- Run `node dist/packages/wp-now/cli.js start --blueprint=b.json`
- Observe no errors in the console and the WordPress page opens successfully with the plugin `notification` installed.

`b.json`
```json
{
	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
	"landingPage": "/wp-admin/",
	"preferredVersions": {
		"php": "8.0",
		"wp": "latest"
	},
	"steps": [
		{
			"step": "login",
			"username": "admin",
			"password": "password"
		},
		{
			"step": "installPlugin",
			"pluginZipFile": {
				"resource": "wordpress.org/plugins",
				"slug": "notification"
			},
			"options": {
				"activate": true
			}
		}
	]
}
```
